tests/foreman/ui/test_dashboard.py

```diff
@@ -631,12 +632,22 @@ def test_positive_content_host_subscription_status(self):
                 set_context(session, org=org.name)
                 self.assertTrue(self.dashboard.validate_chss_navigation(
                     'Invalid', u'subscription_status = invalid'))
-                self.assertIsNotNone(self.dashboard.wait_until_element(
-                    common_locators['kt_search_no_results']))
-                self.assertTrue(self.dashboard.validate_chss_navigation(
-                    'Partial', u'subscription_status = partial'))
-                self.assertIsNotNone(self.dashboard.wait_until_element(
-                    common_locators['kt_search_no_results']))
+                if not bz_bug_is_open(1482150):
```

Comment:
Dunno. Currently while you reading the code you'll see more clear code first and only then ugly workaround :)